stylist/context_processors.py

```diff
@@ -31,7 +31,7 @@ def get_custom_styles(request):
         
     try:
         style = Style.objects.filter(site=site).get(enabled=True)
-        if getattr(settings, 'STYLIST_IGNORE_SASS', False):
+        if getattr(settings, 'STYLIST_IGNORE_SASS', True):
```

Comment:
This would be a bit cleaner with the other naming convention and with out so many "nots"